steward/src/commands/keys/cosmos/add.rs

```diff
@@ -6,24 +6,28 @@ use rand_core::OsRng;
 use signatory::FsKeyStore;
 use std::path;
 
+/// Steward keys cosmos add [name]
```

Comment:
The help message will specify the format of the command. These descriptors are more for a documentation-level description of what this command is doing.